src/main/common/filter.h

```diff
@@ -45,6 +47,13 @@ typedef struct biquadFilter_s {
     float x1, x2, y1, y2;
 } biquadFilter_t;
 
+#define BIQUAD_LPF_ORDER_MAX 6
+
+typedef struct biquadFilterSections_s {
+    biquadFilter_t filterState[(BIQUAD_LPF_ORDER_MAX + 1) / 2];   // each section is of second order
```

Comment:
maybe this is a nitpick but I would call this member `biquad`

Reply:
@codecae I can do that. `state` was the other one I thought of. also the name of the union member is a little extravagant, suggestions for that one?

Reply:
e.g. makes `filter->sections` and `&filter->state` in the `cascadedBiquadFilterApply`

Reply:
called it `biquad`. :+1: